build.gradle

```diff
@@ -6,11 +6,11 @@
 buildscript {
 
     ext {
-        opensearch_version = System.getProperty("opensearch.version", "2.0.0-rc1-SNAPSHOT")
```

Comment:
[plugin-descriptor.properties](https://github.com/opensearch-project/performance-analyzer/blob/81d019af51a62d42592905cffc51f9c77cb8aaac/src/main/resources/plugin-descriptor.properties) also needs to be updated.

Reply:
Thanks for pointing it out. Added.